pkg/cmd/version/version.go

```diff
@@ -0,0 +1,28 @@
+package version
+
+import (
+	"fmt"
+
+	"github.com/coreeng/corectl/pkg/cmdutil/config"
+	"github.com/spf13/cobra"
+)
+
+var (
+	version = "dev"
+	commit  = "unknown"
+	date    = "unknown"
+	arch    = "unknown"
+)
+
+func VersionCmd(cfg *config.Config) *cobra.Command {
```

Comment:
nice, do you think we could add one tiny e2e test proving this works? it's trivial command, also not crucial however, would be nice to know if works on the build. What do you think?

Reply:
I've added an integration test, it's a little tricky to test that the values are correct, so I'm just checking with regex that valid values are filled in